spec/spec_helper.rb

```diff
@@ -18,7 +18,3 @@ def input(plugin, &block)
   end
 
 end
-
-RSpec.configure do |c|
-  c.include IrcHelpers
```

Comment:
yep, we have the same problem in ganglia, udp, gelf, etc, specially the ones that redefine the input helper method to have 2 arguments + block instead of 1 argument + block